.github/workflows/scheduled.yml

```diff
@@ -123,14 +123,25 @@ jobs:
         working-directory: ${{ github.workspace }}
         env:
           GH_TOKEN: ${{ github.token }}
+          INPUT_REF: ${{ inputs.ref || 'main' }}
+          HEAD_REF: ${{ github.head_ref }}
+          BASE_REF: ${{ github.base_ref }}
+          ACTOR: ${{ github.actor }}
         id: get-head
```

Comment:
Should we change name to 'Get merge base from main' also maybe id can be changed to 'get-merge-base' ?